osgp/platform/osgp-adapter-domain-core/src/test/java/org/opensmartgridplatform/adapter/domain/core/application/services/DefaultDeviceResponseServiceTest.java

```diff
@@ -0,0 +1,143 @@
+/**
+ * Copyright 2020 Alliander N.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package org.opensmartgridplatform.adapter.domain.core.application.services;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.verify;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.opensmartgridplatform.adapter.domain.core.infra.jms.ws.WebServiceResponseMessageSender;
+import org.opensmartgridplatform.shared.exceptionhandling.ComponentType;
+import org.opensmartgridplatform.shared.exceptionhandling.OsgpException;
+import org.opensmartgridplatform.shared.exceptionhandling.TechnicalException;
+import org.opensmartgridplatform.shared.infra.jms.CorrelationIds;
+import org.opensmartgridplatform.shared.infra.jms.ResponseMessage;
+import org.opensmartgridplatform.shared.infra.jms.ResponseMessageResultType;
+
+public class DefaultDeviceResponseServiceTest {
+
+    @Mock
+    private WebServiceResponseMessageSender webServiceResponseMessageSender;
+    
+    @InjectMocks
+    private DefaultDeviceResponseService defaultDeviceResponseService;
+
+	//constants that are used in all the tests
+	private static final CorrelationIds ids = new CorrelationIds("orginazationTestId", "deviceIdTest", "correlationUid");
+	private static final String messageType = "Warning";
+	private static final int messagePriority = 3;
```

Comment:
Constants should be all uppercase with words separated by underscores, see https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html

Reply:
These should now be changed to upercase words separated by underscores